tensorflow/c/experimental/gradients/grad_test_helper.cc

```diff
@@ -105,6 +106,36 @@ void CheckTensorValue(AbstractTensorHandle* t, absl::Span<const float> manuals,
   delete[] danalytical;
 }
 
+Model BuildGradModel(Model ops, size_t num_inputs, string ops_name,
+                     GradientFunctionFactory gradient_function_factory) {
```

Comment:
The current signature restricts the usage of `BuildGradModel` to models containing a single op. A more general API might be `BuildGradModel(Model forward, const GradientRegistry& registry)`.

Reply:
Done